src/routes/accounts.py

```
@@ -254,6 +259,76 @@
     )
 
 
+@router.post("/regenerate-activation-link/")
+async def regenerate_activation_link(
+    regenerate_data: GenerateActivationLinkRequestSchema,
+    db: AsyncSession = Depends(get_db),
```

Ruff (0.11.9)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)